### ISSUE TITLE
RenderAsUrl as public method

### ISSUE THREAD

**Issue by gaevoy** [KevReed/PlantUml.Net#11]
I need to have global URL of my diagram and with the help of `http://www.plantuml.com/plantuml/...` it is possible, moreover it is already implemented but hidden via `internal` or `private` modifiers.

It would be nice to have `RenderAsUrl` as method of `IPlantUmlRenderer` or `RemotePlantUmlRenderer` to be able to construct URL of diagram.

Currently, it is possible only via reflection (I use here `ExposedObject` package to do reflection)
```c#
private static Uri RenderPngDiagram(IEnumerable<string> planUmlCode)
{
    var code = string.Join("\n", planUmlCode);
    var renderer = Exposed.From(new RendererFactory().CreateRenderer());
    var urlComponent = (string) renderer.GetUrlComponent(code);
    var urlFormatMap = Exposed.From(renderer.urlFormatMap);
    return new Uri(urlFormatMap.GetRenderUrl(urlComponent, OutputFormat.Png));
}
```

Ideally, it should be something like the following
```c#
private static Uri RenderPngDiagram(IEnumerable<string> planUmlCode)
{
    var code = string.Join("\n", planUmlCode);
    return new RendererFactory()
        .CreateRenderer()
        .RenderAsUrl(code, OutputFormat.Png);
}

```

**Comment by KevReed** [comment]
included in [1.2.40](https://www.nuget.org/packages/PlantUml.Net/1.2.40)

**Comment by gaevoy** [comment]
Great! Thanks for fast feedback. I blog posted and made use of new shiny `RenderAsUri` here https://gaevoy.com/2019/02/14/self-documented-state-machine.html
